x-pack/legacy/plugins/reporting/export_types/common/execute_job/get_custom_logo.ts

```diff
@@ -25,6 +25,16 @@ export const getCustomLogo = async ({
     // We use the basePath from the saved job, which we'll have post spaces being implemented;
     // or we use the server base path, which uses the default space
     getBasePath: () => job.basePath || serverBasePath,
+    path: '/',
```

Comment:
__note:__ this is the ugly hack copied from [other plugin](https://github.com/elastic/kibana/blob/a901d7e5281a07564bcc5a71382e80a33d2b4768/x-pack/legacy/plugins/actions/server/lib/task_runner_factory.ts#L68). There are two points why I still prefer to have it though:
* It allows Security plugin to completely migrate to `KibanaRequest`, convert `LegacyRequest` to `KibanaRequest` only at the LP <-> NP boundaries and completely get rid of dependency on `FakeRequest`
* The `fakeRequest` is already super hacky, even without this change

Reply:
Yeah, I know :disappointed: But I'd rather be radical and fail if we have another place where we provide less "advanced" fake request or if it happens that these set of params isn't enough to construct `KibanaRequest` anymore. It _feels_ this way we can get rid of this fake/legacy requests earlier.

The major risk here is that if we have some code that conditionally constructs fake requests that can't be converted into `KibanaRequest`. And if there is no enough test coverage the problem may manifest itself only in production. To partially mitigate this problem and since [core still supports FakeRequests](https://github.com/elastic/kibana/blob/master/src/core/server/elasticsearch/cluster_client.ts#L86) I'm wondering if core could expose utility to construct fake requests that's fully compatible with its very own `KibanaRequest.from` and protected by the unit test. @elastic/kibana-platform what do you think?